WordPress/Classes/ViewRelated/Jetpack/Social/JetpackSocialNoConnectionView.swift

```diff
@@ -0,0 +1,87 @@
+import SwiftUI
+
+struct JetpackSocialNoConnectionView: View {
+
+    private let viewModel: JetpackSocialNoConnectionViewModel
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: 12.0) {
+            HStack(spacing: -6.0) {
+                iconImage("icon-tumblr")
+                iconImage("icon-facebook")
+                iconImage("icon-linkedin")
+            }
+            Text(Constants.bodyText)
+                .font(.callout)
+            HStack {
+                Text(Constants.connectText)
+                    .font(.callout)
+                    .foregroundColor(Color(UIColor.primary))
+                    .onTapGesture {
+                        viewModel.onConnectTap?()
+                    }
+                if !viewModel.hideNotNow {
+                    Spacer()
+                    Text(Constants.notNowText)
+                        .font(.callout)
+                        .foregroundColor(Color(UIColor.primary))
+                        .onTapGesture {
+                            viewModel.onNotNowTap?()
+                        }
+                }
+            }
+        }
+        .padding(viewModel.padding)
+        .background(Color(UIColor.listForeground))
+    }
+
+    func iconImage(_ image: String) -> some View {
+       Image(image)
+            .resizable()
+            .frame(width: 32.0, height: 32.0)
```

Comment:
Should we add `.clipShape(Circle())` to force any image here to be displayed in a circular container? I'm not sure if we're going to fall back to the icon URL returned from the `external-services` API, but the images returned from there are square-shaped. More context: p1686079466982639-slack-C05362VSXFU

Reply:
Sure, I can add that 👍. It's in: https://github.com/wordpress-mobile/WordPress-iOS/pull/20808/commits/af1c70a42384a06c897974699259a2a97021b2d5